src/libs/OptionsListUtils.ts

```diff
@@ -1931,8 +1939,18 @@ function getOptions(
             return;
         }
 
-        if ((!accountIDs || accountIDs.length === 0) && !isChatRoom) {
-            return;
+        // Reports that aren't searchable via participant won't have any participants returned with SearchForReports
+        // so we don't need to check if their participants exist in order for them to qualify as a valid option.
+        if (isSearchingForReports && !ReportUtils.isSearchableViaParticipants(report)) {
+            return option;
+        }
+
+        if (!accountIDs || accountIDs.length === 0) {
+            if (!isSearchingForReports) {
+                if (!isChatRoom) {
+                    return;
+                }
```

Comment:
NAB:
```suggestion
            if (!isSearchingForReports && !isChatRoom) {
                return;
            }
```

Reply:
Ah shoot, I thought I pushed it but something with my sshkey was borked, my bad for not realizing